### ISSUE TITLE
Search by coin symbol

### ISSUE THREAD

**Issue by shaimo** [graphprotocol/mission-control-curator#352]
Seems like it's impossible to search by coin currently (e.g. SNX doesn't bring up Synthetix). I think each project should list all relevant coins and this info becomes searchable too.
See also this: https://github.com/graphprotocol/mission-control-curator/issues/350

**Comment by yanivtal** [comment]
Yes that would be nice. We should have a separate **Assets** registry that's linked to the **Projects**. We would have to augment the search in the UI to look at the Assets too but that makes sense.
